pkg/daemon/containernetwork/utils.go

```diff
@@ -152,26 +152,22 @@ func checkPodNetConfigReady(podIP net.IP, podCidr *net.IPNet, forwardNodeIfIndex
 			if err != nil {
 				return fmt.Errorf("failed to check cidr %v rule and default route exist: %v", podCidr, err)
 			}
+			defaultRouteExist, err := daemonutils.CheckDefaultRouteExist(table, family)
```

Comment:
Why change the dependencies of these checks? I mean, if the first check failed, there is no need to check the others.

Reply:
Because there will be 4 layers of nested `if` block, which seems not clean enough. And it's fine to always run all the checks, none of them will fail with an error easily.

Reply:
Fixed. 

For the rule check and default route check, they do have a dependency. But for the others, they have no dependencies and will not fail with an error easily. I do this to reduce the number of nested `if` blocks.